zebra-chain/src/parameters/network/subsidy.rs

```diff
@@ -50,6 +50,11 @@ pub const POST_BLOSSOM_HALVING_INTERVAL: HeightDiff =
 /// [7.10.1]: https://zips.z.cash/protocol/protocol.pdf#zip214fundingstreams
 pub const FIRST_HALVING_TESTNET: Height = Height(1_116_000);
 
+/// The first halving height in the regtest is at block height `114`
+/// as [zcashd regtest halving interval](https://github.com/zcash/zcash/blob/v5.10.0/src/consensus/params.h#L252)
+/// is 114 blocks.
+pub const FIRST_HALVING_REGTEST: Height = Height(114);
```

Comment:
The first halving height should depend on the Blossom activation height on Regtest (between 144-287), and also the slow start and halving intervals on custom Testnets, so Zebra can't calculate the halving heights until after it has read its configuration:

![image](https://github.com/user-attachments/assets/f03c0410-2af5-473c-a0ee-7eafe9679602)

(page 134 of the [protocol specification](https://zips.z.cash/protocol/protocol.pdf))

It _could_ use a `OnceCell` for custom Testnets/Regtest and calculate them at compile time for Mainnet and the default Testnet (or use hard-coded values), but it should be quick enough that there's still lower-hanging fruit for improving performance.